dummy.txt

```diff
@@ -0,0 +1,2 @@
+Dummy
```

Comment:
Delete dummy.txt

Reply:
How do I do this also

Reply:
You added the dummy.txt as apart of this pull request. Just go ahead and make another commit deleting it.